dev/ci/ci-hott.sh

```diff
@@ -5,6 +5,5 @@ ci_dir="$(dirname "$0")"
 
 git_download hott
 
-( cd "${CI_BUILD_DIR}/hott" && ./autogen.sh -skip-submodules && ./configure \
-      && sed -i.bak 's/\(HOQC =.*\)/\1 -native-compiler no/' Makefile \
+( cd "${CI_BUILD_DIR}/hott" && coq_makefile -f _CoqProject -o Makefile
```

Comment:
```suggestion
( cd "${CI_BUILD_DIR}/hott" && coq_makefile -f _CoqProject -o Makefile \
```
But I think also this may fail unless the native compiler is disabled?

Reply:
Ah I didn't know about that.

Reply:
Let's try deactivating it too

Reply:
See https://github.com/HoTT/HoTT/issues/1454.  Idk if this is an issue with the native complication scheme, or if it's an issue with having so many things `Defined`